openedx/features/partners/helpers.py

```diff
@@ -107,3 +109,13 @@ def get_partner_from_user(user):
 
 def user_has_performance_access(user, partner):
     return bool(partner and user.has_perm('partners.%s' % PERFORMANCE_PERM_FRMT.format(slug=partner.slug)))
+
+
+def get_partner_registration_countries(partner_slug):
+    sorted_countries = sorted(COUNTRIES.items(), key=operator.itemgetter(1))
+    top_registration_countries = PARTNERS_TOP_REGISTRATION_COUNTRIES.get(partner_slug, None)
+    if top_registration_countries:
+        top_countries = [(country[0], COUNTRIES.get(country[0], None)) for country in top_registration_countries]
+        return {'all_countries': sorted_countries, 'top_countries': top_countries}
+    else:
+        return {'all_countries': sorted_countries, 'top_countries': []}
```

Comment:
it is again related to coding style you can leave it if you want.

```
def get_partner_registration_countries(partner_slug):
    sorted_countries = sorted(COUNTRIES.items(), key=operator.itemgetter(1))
    countries_dict = {'all_countries': sorted_countries, 'top_countries': []}
    top_registration_countries = PARTNERS_TOP_REGISTRATION_COUNTRIES.get(partner_slug, None)
    if top_registration_countries:
        countries_dict.update({'top_countries': top_registration_countries})
    return countries_dict
```

Reply:
I did that initially, but was not sure if I should be updating a pre-made dictionary, when I can just return one based on the case.

Reply:
understood, it is related to coding style totally up to you.